README.md

```diff
@@ -160,6 +160,8 @@ osu-wine --fixsteamdeck: Reinstalls game dependencies after SteamOS updates
 
 Since osu! runs on Wine, you can play that on Steam Deck as well!
 
+It is recommended to not manually install pipewire on the Steam Deck as it is already installed by default and attempting to do so may cause audio issues in gaming mode.
```

Comment:
Doesn't the steam deck update PipeWire just like a normal Arch Linux system tho? Reinstalling would just mean updating the package. Anyways I guess we can add something like this, but worded way shorter.

Reply:
I added that because the first time I installed pipewire-pluse with pacman and it totally killed my microphones on the deck and I had to reset it to get it back working

Reply:
Nice then, I’ll make sure to add it 👌